pkg/internal/meta.go

```diff
@@ -0,0 +1,13 @@
+package internal
+
+const (
+	// lastDeployedReleaseVersion is the version annotation put into Cluster CR to
+	// define which Giant Swarm release version was last successfully deployed
+	// during cluster creation or upgrade. Versions are defined as semver version
+	// without the "v" prefix, e.g. 14.1.0, which means that cluster was created
+	// with or upgraded to Giant Swarm release v14.1.0.
+	LastDeployedReleaseVersion = "release.giantswarm.io/last-deployed-version"
+
+	// UpgradingToNodePools is set to True during the first cluster upgrade to node pools release.
+	UpgradingToNodePools = "release.giantswarm.io/upgrading-to-node-pools"
```

Comment:
Did not want to pull in entire `apiextensions` just for few labels, `UpgradingToNodePools` will be deleted anyway when all clusters are upgraded.

We can discuss the ideas about sharing labels and annotations in different projects, as IMO pulling in `apiextensions` every time in every project in not very efficient, as it's creating more work around dependencies.